Update Sync limitations (#451)
obsidianmd/obsidian-help: en/Obsidian Sync/Limitations.md

## How long do you keep my data after my subscription expires?

> We keep data in your remote vaults, including version history, for one month after your subscription expires. Your local vaults, stored on your devices, will not be affected.

Reviewer: - Remove passive voice and tighten sentence.

> As long as you renew within one month, there should be no impact on your usage. In case you don't and your remote vaults no longer exist, you can easily [[Set up Obsidian Sync|create a new remote vault and re-connect your local vault]] again.

Reviewer: ```suggestion
As long as you renew within one month, there should be no impact on your usage. If you renew after a month, when your remote vaults have been removed, you can [[Set up Obsidian Sync|create a new remote vault re-connect your local vault]].
```

Reviewer: - Change from negative to positive statement.
- "again" is redundant with "RE-connect".

## How long do you keep my data after my subscription expires?

We keep data in your remote vaults, including version history, for one month after your subscription expires.
Your local vaults, stored on your devices, will not be impacted.

> 

Reviewer 2: ```suggestion
In case you don't and your remote vaults no longer exist, you can easily [[Set up Obsidian Sync|create a new remote vault and re-connect your local vault]] again.
```
